app/src/main/java/ch/epfl/sweng/SDP/home/HomeActivity.java

```diff
@@ -498,5 +539,17 @@ public void onResume() {
         ((TextView) findViewById(R.id.starsCount)).setText(String.valueOf(account.getStars()));
         ((TextView) findViewById(R.id.trophiesCount)).setText(String.valueOf(
                 account.getTrophies()));
+        setGameButtons(true);
```

Comment:
This is not really needed here right?

Reply:
@Claudio5 what do you think?

Reply:
Yes it is needed, otherwise the game buttons could be locked if you leave the app on the waiting room and reopen GyroDraw.